nearcore/src/runtime/mod.rs

```diff
@@ -3199,4 +3201,27 @@ mod test {
         let view_state_value = view_state.get(&key).unwrap().unwrap();
         assert_eq!(state_value, view_state_value);
     }
+
+    /// Check that mainnet genesis hash still matches.
+    #[test]
+    fn test_genesis_hash() {
+        let genesis = Genesis::from_file("res/mainnet_genesis.json", GenesisValidationMode::Full);
+        let chain_genesis = ChainGenesis::new(&genesis);
+        let (tempdir, opener) = near_store::NodeStorage::test_opener();
+        let store = opener.open().unwrap();
```

Comment:
Can we use in-memory store? Feels like this test shouldn't write to disk.

Reply:
You don’t even need full NodeStorage. `near_store::test_utils::create_test_store` will give you Store you can pass to runtime constructor.

Reply:
thanks - done.